go.mod

```diff
@@ -1,17 +1,16 @@
 module github.com/mezmo/terraform-provider-mezmo
 
-go 1.20
+go 1.21
```

Comment:
Make sure to rebase before changing this file.  There's been lots of dep changes on main.